pkg/trace/filters/replacer.go

```diff
@@ -24,29 +26,28 @@ func NewReplacer(rules []*config.ReplaceRule) *Replacer {
 }
 
 // Replace replaces all tags matching the Replacer's rules.
-func (f Replacer) Replace(trace pb.Trace) {
+func (f Replacer) Replace(_ *api.Metadata, _ *traceutil.ProcessedTrace, s *pb.Span) error {
```

Comment:
(here and throughout) I'm almost definitely missing something here, but why have blank parameters instead of just doing a blank import for both of those library if you are counting on side effects?

Reply:
Ah I see why now. It's to support a generic interface for these functions. Seems reasonable! (I'll think on it for a bit and let you know if there's a different way that might be cleaner, but definitely don't block on me)